spec/models/user_spec.rb

```diff
@@ -7,21 +7,21 @@
     it { should have_many(:ips) }
```

Comment:
Could add `it { should validate_presence_of(:name).on(:update) }`

Reply:
👍